gtsam/slam/GeneralSFMFactor.h

```diff
@@ -70,6 +70,7 @@ class GeneralSFMFactor: public NoiseModelFactor2<CAMERA, LANDMARK> {
 protected:
 
   Point2 measured_; ///< the 2D measurement
+  bool verbose_; ///< Flag for print verbosity
```

Comment:
Adding a boolean flag for 100k factors in an SFM problem. Problematic. Propose to leave this out of this PR and discuss a better strategy

Reply:
Alright. Should I leave the exception printing in? Ideally that should be part of a logging framework. I'll already looked up some options.

Reply:
No, no printing. Let's leave for when we have logging, which would be great. I do like glog :-)

Reply:
Yeah glog and CodeCov (for unit test code coverage) are two things on my radar. I'll create an issue for it that someone else can tackle.